packages/editor/src/components/browser/function/FunctionBrowser.tsx

```diff
@@ -36,37 +36,32 @@ const FunctionBrowser = (props: { value: string; onChange: (value: BrowserValue)
   const [method, setMethod] = useState('');
   const [paramTypes, setParamTypes] = useState<string[]>([]);
   const [type, setType] = useState('');
-  const [sortedTree, setSortedTree] = useState<Function[]>([]);
   const { data: tree } = useMeta('meta/scripting/functions', undefined, []);
   const { data: doc } = useMeta('meta/scripting/apiDoc', { context, method, paramTypes, type }, '');
+  const sortedTree = useMemo(() => {
+    if (!tree || tree.length === 0) {
+      return [];
```

Comment:
have an initial tree here doesn't work with the init expanded state + virtualize?

Reply:
No, unfortunately not 😬